pkg/pillar/base/execwrapper.go

```diff
@@ -134,6 +97,19 @@ func (c *Command) execCommand() ([]byte, error) {
 	}
 }
 
+// WithCustomTimeout set custom timeout for command
+func (c *Command) WithCustomTimeout(timeout time.Duration) *Command {
+	// $ grep hv_watchdog_timer pkg/grub/rootfs.cfg
+	const watchdogTimeout = 500 * time.Second
```

Comment:
Hmmm... As far as I understand, each service has its own warning/error timeout. It's defined at the moment the `StillRunning` functions are called in each component.
Or are we talking about another watchdog?

Reply:
You're right, I updated the comment to reflect the correct watchdog.
Seems they're all configured to 3 minutes (also updated).

Reply:
from the logfile: `watchdog: file /run/zedagentdevinfo.touch was not changed in 538 seconds (more than 500)`
and that seems to come from `/etc/watchdog.conf`:
```
watchdog-device = /dev/watchdog
file = /run/zedagent-localappinfo.touch
change = 500
file = /run/volumemgrmetrics.touch
change = 500
file = /run/zedagentccerts.touch
change = 500
file = /run/zedagentecerts.touch
change = 500
file = /run/zedagentattest.touch
change = 500
file = /run/zedagentconfig.touch
change = 500
...
```